src/webapp/features/join/index.tsx

```diff
@@ -136,14 +137,15 @@ const JoinGame = () => {
                 name="nickname"
```

Comment:
While we're in here, mind fixing the CSS for the label so it looks better on IOS?

<img width="354" alt="ios input label padding" src="https://user-images.githubusercontent.com/4634442/105638794-974ce080-5e29-11eb-8ad9-0bd5980402ae.png">

Reply:
Like 1-2px padding is what i'm thinking